{{cookiecutter.project_slug}}/setup.cfg

```diff
@@ -19,5 +19,5 @@ exclude = docs
 
 {%- if cookiecutter.use_pytest == 'y' %}
 [tool:pytest]
-collect_ignore = ['setup.py']
+addopts = --ignore=setup.py
 {%- endif %}
```

Comment:
**Suggestion:** Consider adding a comment to explain why 'setup.py' is being ignored. This will make the code more understandable for other developers.
```suggestion
{%- if cookiecutter.use_pytest == 'y' %}
[tool:pytest]
# Ignore setup.py as it's not part of the test suite
addopts = --ignore=setup.py
{%- endif %}
```